### DISCUSSION TITLE
Change color of already added category name in selection box (entries)

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#8420] [Ideas]
### Description
The color of already added category name is poorly visible.
![category](https://cloud.githubusercontent.com/assets/18059922/24923891/41b62fa6-1ef3-11e7-9cf0-2c2d61566383.png)

### Steps to reproduce

1. Try to add a category.
2. Voila!

### Additional info

- Craft version: Craft CMS 2.6.2971

**Comment by brandonkelly** [comment]
Not sure I'm following. Can you elaborate a bit?

**Comment by ghost** [comment]
@brandonkelly 
I mean that this text should be darker, as it is poorly visible
![category](https://cloud.githubusercontent.com/assets/18059922/24926187/0d64ab08-1efb-11e7-9607-a29f816346be.png)
